26-register_registration/creator_begins_registering_a_registration.feature

```diff
@@ -23,47 +34,55 @@ Feature: Creator begins registering a Registration
+    And they expand the Expansion panel for Link to resource
     And they enter a DOI or a fully qualified DOI URL
     And they click Search
     Then they see metadata about the Link in the Expansion panel
 
   @439
   Scenario: Creator begins registering with a Link with data from Datacite/Crossref from citation_doi meta tag (DOI)
     Given Creator begins registering a Registration
-    And they click Expansion panel for Link to resource
+    And they expand the Expansion panel for Link to resource
     And they enter https://dlr.unit.no/resources/66888570-3504-4d12-81a4-c3ffe0605945
     And they click Search
     Then they see metadata about the Link in the Expansion panel
 
   @440
   Scenario: Creator begins registering with a Link with data from dc:identifier meta tag
     Given Creator begins registering a Registration
-    And they click Expansion panel for Link to resource
+    And they expand the Expansion panel for Link to resource
     And they enter https://loar.kb.dk/handle/1902/1674?show=full
     And they click Search
     Then they see metadata about the Link in the Expansion panel
 
   @441
   Scenario: Creator begins registering with a Link with data from DC and DCTERMS meta tags
     Given Creator begins registering a Registration
-    And they click Expansion panel for Link to resource
+    And they expand the Expansion panel for Link to resource
     And they enter https://ntnuopen.ntnu.no/ntnu-xmlui/handle/11250/2638973
     And they click Search
     Then they see metadata about the Link in the Expansion panel
 
   @442
   Scenario: Creator begins registering with a Link with data from Open Graph tag
     Given Creator begins registering a Registration
-    And they click Expansion panel for Link to resource
+    And they expand the Expansion panel for Link to resource
     And they enter https://www.nrk.no/norge/klimakur-2030_-mer-strom-og-mindre-kjott-kan-fa-norge-i-mal-1.14883788
     And they click Search
     Then they see metadata about the Link in the Expansion panel
 
   @2208
   Scenario: Creator begins registering with a Link with data from Highwire tag
     Given Creator begins registering a Registration
-    And they click Expansion panel for Link to resource
+    And they expand the Expansion panel for Link to resource
     When they enter https://link.springer.com/article/10.1007/s13201-020-01350-9
     And click Search
-    Then they see metadata about the Link in the Expansion panel
\ No newline at end of file
+    Then they see metadata about the Link in the Expansion panel
+
+  @2370
+  Scenario: Creator begins registering with a Link with data from LD+/JSON tag
+    Given Creator begins registering a Registration
+    And they expand the Expansion panel for Link to resource
+    When they enter "https://e24.no/boers-og-finans/i/AdyrPM/syv-av-tangens-11-referanser-deltok-paa-luksusseminaret"
+    And click Search
+    Then they see metadata about the Link in the Expansion panel
```

Comment:
Her sliter vi litt siden vi ikke helt vet hvilke metadata vi ønsker ut og hvilke mappinger vi ønsker, men jeg antar at vi beskriver dette i en annen oppgave?

Reply:
Godt spørsmål. Dette Scenarioet beskriver hva brukeren skal se etter søk, og vet ikke hvor enkelt det er å teste alle mappinger  som ikke skal vises her, så selve testing av det passer kanskje bedre i API-tester?
I så fall kunne vi egentlig slått sammen mange av disse Scenarioene til `When they enter a link`, så har vi API-tester for mapping av de forskjellige typene

Reply:
```suggestion
    Then they see Metadata about the Link in the Expansion panel
```

Foreslår å skrive `Metadata` som konsept, og legge til scenario for hver av de forskjellige typene med hvilke metadata som skal være med

Reply:
Ser bare ikke helt hvordan det skal kunne testes. Vi kan åpne wizard etterpå og sjekke på feks tittel der, men vi aner ikke om den tittelen kommer fra `schema.org`, `highwire`, eller en av de 5 andre scenarioene som ligger over her. Dermed kan testen for `schema.org` gå gjennom helt fint, men så kommer egentlig all data fra en annen plass 🤔

Reply:
åja, sånn ja. Ja, men da burde det kanskje heller være en del av API-testingen da